applications/openshift/master/file_groupowner_kube_apiserver/rule.yml

```diff
@@ -1,15 +1,15 @@
 documentation_complete: true
 
-prodtype: ocp4
+prodtype: rhcos4
 
 title: 'Verify Group Who Owns The Kube API Server Pod Specification File'
 
-description: '{{{ describe_file_group_owner(file="/etc/kubernetes/static-pod-resources/kube-apiserver-pod.yaml", group="root") }}}'
+description: '{{{ describe_general_file_group_owner(filedescription="API server pod specification", group="root") }}}'
```

Comment:
The description needs to use the original macro and needs to provide guidance as to what file permissions need to be checked and verified on the system to the file. Otherwise, there is no way to read or understand what you are supposed to do.

Reply:
BTW... these type of changes is what an overlay file (industry concept/standard) is for.